x-pack/plugins/stack_alerts/server/alert_types/index_threshold/alert_type.ts

```diff
@@ -83,6 +83,20 @@ export function getAlertType(service: Service): AlertType<Params, {}, {}, Action
     }
   );
 
+  const actionVariableContextFunctionLabel = i18n.translate(
+    'xpack.stackAlerts.indexThreshold.actionVariableContextFunctionLabel',
+    {
+      defaultMessage: 'A string formatted values combining threshold comparator and threshold',
+    }
+  );
+
+  const actionVariableContextWindowLabel = i18n.translate(
+    'xpack.stackAlerts.indexThreshold.actionVariableContextWindowLabel',
+    {
+      defaultMessage: 'A string formatted values combining time window size and time window unit',
```

Comment:
In looking at this, I just realized we shouldn't need to add `window` as a separate context variable.  There is already a context variable containing the parameters, called `alertParams`, so these can be referenced in the message string as 

```
{{alertParams.timeWindowSize}}{{alertParams.timeWindowUnit}}
```

which should yield a string like `5s` (5 seconds), which is all we're doing below anyway.

So we shouldn't need the `window` variable at all.

Note, in theory we might be able to do the `function` variable in the same way, but it would make the mustache template a lot more complicated, not sure it's worth it, and it feels useful to have a nice version of the "function" available easily anyway.